pkg/api/server.go

```diff
@@ -44,7 +44,7 @@ func NewAPIServer(
 	enableReflection bool,
 	messagePublisher blockchain.IBlockchainPublisher,
 ) (*ApiServer, error) {
-	grpcListener, err := net.Listen("tcp", fmt.Sprintf("0.0.0.0:%d", port))
+	grpcListener, err := net.Listen("tcp", fmt.Sprintf("localhost:%d", port))
```

Comment:
This gets rid of those pesky 'Allow incoming connections' dialogue when running unit tests - we only want to listen to incoming connections on localhost

Reply:
Interesting...

Great find